.gitignore

```diff
@@ -13,3 +13,9 @@ test/build/**/web_modules
 test/create-snowpack-app/test-install
 test/esinstall/**/web_modules
 yarn-error.log
+
+# Ignore files that are copied from /docs
+www/_template/concepts/**/*.md
```

Comment:
any reason not to do `www/_template/*/**/*.md` or `www/_template/**/*.md` ?

Reply:
There are still some markdown files in `www/_template/posts/**/*.md`, but an opt-out for those is probably cleaner than this approach